vendor/vendor.json

```diff
@@ -741,17 +741,29 @@
 			"revision": "d77da356e56a7428ad25149ca77381849a6a5232",
 			"revisionTime": "2016-06-15T09:26:46Z"
 		},
+		{
+			"checksumSHA1": "4VppKBbzCSmoFfQxGNUm0TYiFCA=",
+			"path": "github.com/prometheus/client_golang/api",
```

Comment:
Why are you pulling these in? We don't use them.

Reply:
I'm working a particular open task (adding query to promtool) and was hoping to use these for them.

Reply:
This one! https://github.com/prometheus/prometheus/issues/1605